src/internal/methods/NeoEsp32LcdXMethod.h

```diff
@@ -0,0 +1,551 @@
+#pragma once
+
+#if defined(ARDUINO_ARCH_ESP32) && defined(CONFIG_IDF_TARGET_ESP32S3)
+
+extern "C"
+{
+#include <driver/periph_ctrl.h>
+#include <esp_private/gdma.h>
+// #include <rom/gpio.h>
+#include <esp_rom_gpio.h>
+#include <hal/dma_types.h>
+#include <hal/gpio_hal.h>
+#include <soc/lcd_cam_struct.h>
+}
+
+class NeoEspLcdMuxBusSize8Bit
+{
+public:
+    NeoEspLcdMuxBusSize8Bit() {};
+
+    const static size_t MuxBusDataSize = 1;
+
+    static void EncodeIntoDma(uint8_t** dmaBuffer, const uint8_t* data, size_t sizeData, uint8_t muxId)
+    {
+        uint8_t* pDma = *dmaBuffer;
+        const uint8_t* pEnd = data + sizeData;
+
+        for (const uint8_t* pValue = data; pValue < pEnd; pValue++)
+        {
+            uint8_t value = *pValue;
+
+            for (uint8_t bit = 0; bit < 8; bit++)
+            {
+                // Get what's already there (offset 1)
+                uint8_t dmaVal = *(pDma + 1);
+
+                // Adjust
+                dmaVal |= (value & 0x80) ? (0x01 << muxId) : 0x00;
+
+                // Write it back
+                *(pDma++) = 0xFF;
```

Comment:
A bulk memory fill of 0xFF 0x00 0x00 should be done in reset to tighten this loop, TODO